gef.py

```diff
@@ -5058,6 +5058,16 @@ def do_invoke(self, argv):
         needle_sections = []
         haystack_sections = []
 
+        if "0x" in haystack:
+            start = int(haystack.split("-")[0], 16)
```

Comment:
It would be more elegant to create a global function like `parse_string_range()` instead of repeating the same block of code twice (also it could be used in other locations).